Flow.Launcher/Converters/ProgressForegroundConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace Flow.Launcher.Converters
+{
+    public class ProgressForegroundConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            double progress = (double)value;
+            string foreground = "#26a0da";
+
+            if (progress >= 90)
+            {
+                foreground = "#da2626";
+            }
+
+            return foreground;
+        }
```

Comment:
@Garulf @onesounds @taooceros Do we want to enable plugins to use progress bar? If so we will need to make this bar colorization customizable by plugin, otherwise any progress above 90 will be red.

Reply:
Probably best we don't hard code red here and allow the color to be adjusted in some manner. 

Best case scenario it's themed and easily overridden by the plugin.